Vonk.Plugin.ExampleOperation/VonkPluginConformanceContributor.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace Vonk.Plugin.ExampleOperation
 {
+    // The ContextAware attribute lets implementations choose for which information models (e.g. FHIR STU3, R4) the service should be made available
     [ContextAware(InformationModels = new[] { VonkConstants.Model.FhirR3 })]
     internal class VonkPluginConformanceContributor : ICapabilityStatementContributor
```

Comment:
For clarity you might want to rename the class to match the new interface more closely: ``VonkPluginCapabilityStatementContributor`` (agreed, long name, but clear).